requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Add Biryani extra requirement here because it is needed by OpenFisca-Core and installation from Git URL below
 # does not take extras_require of Core setup.py into account.
 Biryani[datetimeconv] >= 0.10.4
---editable git+https://github.com/openfisca/openfisca-core.git#egg=OpenFisca-Core
+--editable git+https://github.com/openfisca/openfisca-core.git@simplify-periods#egg=OpenFisca-Core
```

Comment:
Quand je commit ce genre de chose, j'ai pour habitude de préfixer le commit par un gros [TO REVERT], pour que ce soit plus facilement visible à la review.